aspnetcore/host-and-deploy/docker/includes/building-net-docker-images5.md

```diff
@@ -0,0 +1,350 @@
+:::moniker range="< aspnetcore-7.0"
+
+## ASP.NET Core Docker images
```

Comment:
I'm curious, do we need to copy those docs to this repository?

Reply:
I don't understand the question. We've started moving the older versions to an include file.

Reply:
Never mind. I thought this documentation was from the external link.